src/interfaces/interfaces.ts

```diff
@@ -302,6 +309,7 @@ namespace interfaces {
         whenTargetNamed(name: string | number | symbol): BindingOnSyntax<T>;
         whenTargetIsDefault(): BindingOnSyntax<T>;
         whenTargetTagged(tag: string | number | symbol, value: any): BindingOnSyntax<T>;
+        whenTargetMultiTagged(...tags: [interfaces.Tag, ...interfaces.Tag[]]): BindingOnSyntax<T>;
```

Comment:
```Typescript
(...tags: [interfaces.Tag, ...interfaces.Tag[]])
```

This type forces the presence of at least one tag.

Reply:
Be better if you could force at least two
(...tags: [interfaces.Tag,interfaces.Tag, ...interfaces.Tag[]])